src/conn/conn_api.c

```diff
@@ -1047,22 +1047,13 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
 	const char *ckpt_cfg;
 
 	conn = (WT_CONNECTION_IMPL *)wt_conn;
-	ckpt_cfg = "use_timestamp=false";
 
 	CONNECTION_API_CALL(conn, session, close, config, cfg);
-
-	/* The default session is used to access data handles during close. */
-	F_CLR(session, WT_SESSION_NO_DATA_HANDLES);
+err:
```

Comment:
Having the `err:` label at the start of the function is quite unusual and worthy of a comment.

Reply:
I don't object to a comment if you want, feel free to add what you like.

But, it's not that unusual, we do it everywhere there's a API macro (XXX_API_) and we're not jumping out of the function on error, usually found in some API "close" function )(`__curbackup_close()`, `__curconfig_close()`, `__curds_close()`, `__curdump_close()`, `__curfile_close()`, `__curindex_close()`, `__curjoin_close()`, `__curlog_close()`, `__curmetadata_close()`, `__curstat_close()`, `__curtable_close()`).